controllers/cloud.redhat.com/namespaces.go

```diff
@@ -285,10 +280,40 @@ func DeletePrometheusOperator(ctx context.Context, cl client.Client, nsName stri
 	prometheusOperator.SetGroupVersionKind(gvk)
 	prometheusOperator.SetName(GetPrometheusOperatorName(nsName))
 
+	err := cl.Get(ctx, types.NamespacedName{Name: GetPrometheusOperatorName(nsName)}, &prometheusOperator)
+	if k8serr.IsNotFound(err) {
+		return true, nil
+	} else if err != nil {
```

Comment:
Same kind of thing here. What if `err` is not nil but is also not a `NotFound` err? I think this should just be a plain `else {` statement.

Reply:
If you had it as a bare else, then a `nil` value for err would trigger it